### ISSUE TITLE
Uncaught ReferenceError

### ISSUE THREAD

**Issue by petuhovskiy** [BestStartapers2018/game1#3]
```
Uncaught ReferenceError: timeToDieForStar is not defined
    at createStar (main.js:241)
    at updateScore (main.js:354)
    at tryToCreate (main.js:280)
    at Object.update (main.js:340)
    at i.StateManager.update (phaser.js:29361)
    at i.Game.updateLogic (phaser.js:36175)
    at i.Game.update (phaser.js:36114)
    at i.RequestAnimationFrame.updateRAF (phaser.js:64223)
    at window.requestAnimationFrame.forceSetTimeOut._onLoop (phaser.js:64206)
```

**Comment by petuhovskiy** [comment]
@DenisAltruist исправляй

**Comment by petuhovskiy** [comment]
![image](https://user-images.githubusercontent.com/8150127/38145554-8b532824-3452-11e8-8335-0bce279c2bef.png)

**Comment by petuhovskiy** [comment]
Fixed in #1
